docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -158,6 +158,13 @@ Envoy uses the final XFF contents to determine whether a request originated exte
 internally. This influences whether the :ref:`config_http_conn_man_headers_x-envoy-internal` header
 is set.
 
+To test IPv6 compatibility of upstream services which parse XFF header values,
```

Comment:
Playing devils advocate, why can't a real IPv6 connection from the client just be used here?

Reply:
I will let @danielhochman provide his own take, but my opinion is that actually testing with real client IPv6 is a huge operational burden which typically involves LB changes, DNS changes, very hard to roll forward and roll back, etc. IMO this is a very lightweight way for people to flush out issues when beginning an IPv4->IPv6 transition, as a large part of the Internet is about to do. :)

Reply:
Yeah, sure, makes sense, thanks for the explanation.

Reply:
yep, i'll add a sentence to that effect.